lib/ember_cli/path_set.rb

```diff
@@ -148,6 +148,18 @@ def yarn?
       app_options[:yarn] || app_options[:yarn_path]
     end
 
+    def workspace?
+      app_options[:workspace]
```

Comment:
Actually I just realized when looking at the other PR -- can this instead be inferred by the fact that there's a set of `workspaces` in the `package.json` file?

Reply:
Yep, that should be possible. I think we'll have to traverse up the folder structure and check if a package.json exists. Since nested workspaces aren't possible, we can stop there. If one exists, we'll have to check if any of the paths under `workspaces` matches, since a repo can have both workspace and non-workspace projects. Do you think we should try this approach instead?

Reply:
@simpsonjoshua eh since this is a rails specific gem I think you can simply check the `package.json` at `Rails.root` and that's good enough. We're not going to be doing nested workspaces, right?

Reply:
Alright, I gave this a shot: e17e947a9543b7e55ada7a18c4bcc9eb4bb0fcc5